Fence and Coordinates

kristianmatzen opened the pull request:
Added Fence. Coordinates are number instead of strings

kristianmatzen committed:
Fence and Coordinates

Added Fence. Coordinates are number instead of strings

loklaan commented:
The fence will be an array of lng/lat, ie [Polygon](https://developers.google.com/maps/documentation/javascript/reference#Polygon)

loklaan committed:
Resolves ST-89

Known issue: Locations and Fences belonging to a patient
are not destroyed when that patient record is destroyed.

loklaan commented:
I sign off on this, will wait for someone else to merge it.
